docs/source/optimisation.rst

```diff
@@ -64,7 +63,8 @@ print to screen of the status of the optimisation.
    :members:
    :special-members:
 .. autoclass:: cil.optimisation.algorithms.PDHG
-   :members:
+   :members: update, pdhg_step_sizes, update_step_sizes, update_objective
```

Comment:
if we use `_` to mark internal methods then auto-class will not add them to the docs and it's simpler to maintain.